tasks/miscellaneous-debian.yml

```diff
@@ -10,3 +10,25 @@
     name=ufw
     enabled=false
   when: disable_firewall
+
+- name: Check current timezone
+  shell: cat /etc/timezone
```

Comment:
prefer using `command` instead of `shell` (more secure) see http://docs.ansible.com/ansible/shell_module.html / http://docs.ansible.com/ansible/command_module.html